container/bin/entrypoint.py

```diff
@@ -0,0 +1,75 @@
+#!/venv/bin/python3
+#
+# SPDX-FileCopyrightText: Red Hat, Inc.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+import argparse
+import json
+import os
+import subprocess
+
+from ovirt_imageio.client import info
+from ovirt_imageio.client._api import _tmp_dir
```

Comment:
Importing _tmp_dir from _api is ugly. We should not import private names!

To correct this, move _api._tmp_dir to another module as public name, like _util.tmp_dir.